src/index.html

```diff
@@ -13,6 +13,55 @@
     />
   </head>
   <body>
-    <h1>Stars</h1>
+    <main>
+      <div class="stars stars--0">
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
+        <div class="stars__star"></div>
```

Comment:
Remember to add an empty line between multiline sibling blocks of HTML. It will add some 'air' and simplify reading. But don't add them between parent and child elements.